Formula/salt.rb

```diff
@@ -47,15 +48,16 @@ class Salt < Formula
     sha256 "f991347f5b11589ac8dc5a3c8257a514cf802545b75c11133a43ae9f76388278"
   end
 
+  # Fix loading of unversioned /usr/lib/libcrypto.dylib
+  # Remove when merged or https://github.com/saltstack/salt/issues/55084 fixed
+  patch do
+    url "https://github.com/saltstack/salt/pull/56958.patch?full_index=1"
```

Comment:
Maybe a bit brittle in case the PR is updated? How about https://github.com/saltstack/salt/commit/3dea0e31759b6c2a2c7b46647827a72f7a20dafd.patch

Reply:
> Maybe a bit brittle in case the PR is updated? How about https://github.com/saltstack/salt/commit/3dea0e31759b6c2a2c7b46647827a72f7a20dafd.patch

If PR will be rebased this commit _probably_ disappear as well.
When we have bottles for these, changes in PR won't affect major amount of installations.

Generally we're not in favour of using patches from unmerged PR (since a lot of things could go wrong). But taking into account this bug exists for a half a year, we could make a small exception I believe

Reply:
I prefer to point to the PR. If that one is rebased it'll still contain a reference to the work, while the commit will just lose all context

Reply:
> If PR will be rebased this commit _probably_ disappear as well.

It won't.

I think it would be good to link to the PR but don't use it as the URL.

Reply:
Yes, IME Github keeps the commit objects around, at least more than long enough.

My concern is given the patch is verified with a hash, any updates to the PR easily make this formula not build anymore.